### ISSUE TITLE
Do Not Remove Outline On Keyboard Focus

### ISSUE THREAD

**Issue by mzlock** [ganapativs/bttn.css#1]
Do not remove the outline on buttons when they gain keyboard focus.  This is an accessibility issue since the outline aids users navigating sites with only their keyboard.

Consider using a tool like what-input if you'd like to remove the outline for mouse users while keeping the outline for keyboard users:
https://github.com/ten1seven/what-input

**Comment by ganapativs** [comment]
Probably its not a good idea to add JS dependency(what-input) to a CSS library, So, Now library doesn't override outline by default, but in case you want to disable outlines, a helper class(`bttn-remove-outline`) is added to remove all outlines on a button.

Does this solve your issue?

**Comment by ganapativs** [comment]
Can I close the issue?

**Comment by mzlock** [comment]
@ganapativs yes, that's a decent compromise, I think.  I appreciate the new accessibility toggle. I would maybe argue that the toggle should be on by default, but I won't harp on it.  Thank you for your work!
